.github/workflows/test-podman-compose.yml

```diff
@@ -0,0 +1,113 @@
+
+name: Test Podman Compose
+
+on:
+  schedule:  # UTC at 0600
+      - cron:  '0 6 * * *'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  ANSYSLMD_LICENSE_FILE: ${{ format('1055@{0}', secrets.LICENSE_SERVER) }}
+  PYFLUENT_TIMEOUT_FORCE_EXIT: 30
+  PYFLUENT_LAUNCH_CONTAINER: 1
+  PYFLUENT_USE_PODMAN_COMPOSE: 1
+  PYFLUENT_LOGGING: 'DEBUG'
+  PYFLUENT_WATCHDOG_DEBUG: 'OFF'
+  PYFLUENT_HIDE_LOG_SECRETS: 1
+
+jobs:
+  install-podman:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: gacts/install-podman@v1
+      - run: podman compose version
+
+  build_test:
+    name: Build and Test
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
I think testing for only one Fluent version is fine

Reply:
Ok. Then it will help to reduce the overall time significantly. We will use latest release only.

Reply:
Done. Thank you.